lib/manager/helmv3/artifacts.ts

```diff
@@ -10,6 +10,20 @@ import {
 } from '../../util/fs';
 import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
 
+async function helmRepoAdd(alias: string, url: string): Promise<void> {
+  const cmd = `helm repo add ${alias} ${url}`;
+
+  const execOptions: ExecOptions = {
+    docker: {
+      image: 'helm',
+    },
+    extraEnv: {
+      HELM_EXPERIMENTAL_OCI: '1',
+    },
+  };
+  await exec(cmd, execOptions);
```

Comment:
These commands should instead be added to the existing `cmd` (using an array instead of string) so that they are executed sequentially within the same container.